website/docs/configuration/git.md

```diff
@@ -252,6 +252,17 @@ If set to `true`, tags are processed in topological order instead of chronologic
 
 This can also be achieved by using the `--topo-order` command line flag.
 
+### topo_order_commits
+
+If set to `false`, commits are processed in chronological order instead of topological.
```

Comment:
We should use the same wording as `topo_order` above.

```suggestion
If set to `true`, commits are processed in topological order instead of chronological.
```